### PR TITLE
Docs: Add Query history search documentation

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Adds documentation for [Query history search](https://github.com/grafana/grafana/pull/25747), which is a new feature. Also updates documentation for [changed default settings](https://github.com/grafana/grafana/pull/25732).

### CODE REVIEW

File: docs/sources/features/explore/index.md [hunk 1]
```diff
@@ -95,6 +95,13 @@ In **Query history** tab it is also possible to filter queries by date using the
 
 > Note: If you are in split mode, filters are applied only to your currently active panel.
 
+### Search in query history
+
+You can search in your history across queries and your comments. Search is possible in Query history tab and Starred tab.
```
Comment (ivanahuckova) [inline docs/sources/features/explore/index.md:100]:
```suggestion
You can search in your history across queries and your comments. Search is possible for queries in Query history tab and Starred tab.
```

File: docs/sources/features/explore/index.md [hunk 2]
```diff
@@ -95,6 +95,13 @@ In **Query history** tab it is also possible to filter queries by date using the
 
 > Note: If you are in split mode, filters are applied only to your currently active panel.
 
+### Search in query history
+
+You can search in your history across queries and your comments. Search is possible for queries in Query history tab and Starred tab.
```
Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:100]:
```suggestion
You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
```

File: docs/sources/features/explore/index.md [hunk 3]
```diff
@@ -229,12 +237,13 @@ Shows condensed view or the trace timeline. Drag your mouse over the minimap to
...
 
 Shows list of spans within the trace. Each span row consists of these components:
+
 - Expand children button: Expands or collapses all the children spans of selected span.
 - Service name: Name of the service logged the span.
 - Operation name: Name of the operation that this span represents.
 - Span duration bar: Visual representation of the operation duration within the trace.
 
-Clicking anywhere on the span row will show span details. 
+Clicking anywhere on the span row will show span details.
```
Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:246]:
```suggestion
Clicking anywhere on the span row shows span details.
```

**Review by oddlittlebird — APPROVED** [review]
A couple minor edits, then good to go!

File: docs/sources/features/explore/index.md [hunk 4]
```diff
@@ -103,7 +110,7 @@ You can customize the query history in the Settings tab. Options are described i
 | ------------------------------------------------------------- | --------------------------------------- |
 | Period of time for which Grafana will save your query history | 1 week                                  |
 | Change the default active tab                                 | Query history tab                       |
-| Only show queries for datasource currently active in Explore  | False                                   |
+| Only show queries for datasource currently active in Explore  | True                                    |
```
Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:113]:
```suggestion
| Only show queries for data source currently active in Explore  | True                                    |
```

File: docs/sources/features/explore/index.md [hunk 5]
```diff
@@ -95,6 +95,13 @@ In **Query history** tab it is also possible to filter queries by date using the
 
 > Note: If you are in split mode, filters are applied only to your currently active panel.
 
+### Search in query history
+
+You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
+
+1. Click the **Search queries** field.
+2. Type the term you are searching for into search field.
```
Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:103]:
```suggestion
1. Type the term you are searching for into search field.
```

**Review by oddlittlebird — COMMENTED** [review]
A couple minor edits.

File: docs/sources/features/explore/index.md [hunk 6]
```diff
@@ -95,6 +95,13 @@ In **Query history** tab it is also possible to filter queries by date using the
 
 > Note: If you are in split mode, filters are applied only to your currently active panel.
 
+### Search in query history
+
+You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
+
+1. Click the **Search queries** field.
+1. Type the term you are searching for into search field.
```
Comment (ivanahuckova) [inline docs/sources/features/explore/index.md:103]:
```suggestion
2. Type the term you are searching for into search field.
```

File: docs/sources/features/explore/index.md (hunk 5 above)
Comment (oddlittlebird) [reply docs/sources/features/explore/index.md:103]: Per the [Markdown style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md), we want to use lazy numbering by default.

**Pull request merged**